R/structural.R

```diff
@@ -211,7 +211,7 @@ structural <- function(x, transformation, var = character(),
 
     ## iterate through each column and check if the "mass" is in the interval
     ## defined by the m/z value and ppm
-    for (i in seq_along(transformation[, "mass"])) {
+    for (i in seq_len(nrow(transformation))) {
```

Comment:
Hello @tnaake, I observed that the unit tests you already had would fail if the transformation table was a tibble object, so I added a conversion to data.frame at the beginning of the structural function. And I also added the corresponding unit tests.
Thanks!